docker.py

```diff
@@ -157,6 +157,17 @@ def get_numpy_choices():
     return choices
 
 
+def get_scipy_choices():
+    cupy_version = version.get_cupy_version()
+    if cupy_version[0] < 8:
+        choices = [
+            None, '0.18', '0.19', '1.0', '1.1', '1.2', '1.3', '1.4', '1.5']
+    else:
+        # cupy v8 or later
+        choices = ['1.3', '1.4', '1.5']
```

Comment:
Test that SciPy is optional.
```suggestion
        choices = [None, '1.3', '1.4', '1.5']
```